_dev/src/components/product-feed/settings/delivery-time-and-rates/delivery-time-and-rates.vue

```diff
@@ -70,8 +210,15 @@ export default {
       }
       return true;
     },
+    getRate(value) {
+      this.rateChosen = value;
+    },
     previousStep() {
-      localStorage.setItem('productFeed-deliveryDetails', JSON.stringify(this.carriers));
+      if (this.getShippingValueSetup === ShippingSetupOption.IMPORT) {
+        localStorage.setItem('productFeed-customCarrier', JSON.stringify(this.customCarrier));
+      } else {
+        localStorage.setItem('productFeed-deliveryDetails', JSON.stringify(this.carriers));
+      }
```

Comment:
You mpean to save the current changes when going to the previous step? This should not happen, we even warn that changes will be lost